docker/k8s/docker/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM sammyne/hyperledger-tcf-vm:alpha 
+
+LABEL maintainer=lixiangmin01@baidu.com
+
+RUN apt update -y && apt install -y vim
+# the sed cmd below is to decrease the 1TB LMDB cache size for local development
+RUN cd /project && rm -rf TrustedComputeFramework &&\
+    git clone --branch master --depth 1 https://github.com/hyperledger-labs/trusted-compute-framework.git &&\
+    mv trusted-compute-framework TrustedComputeFramework &&\
+    sed -i 's/#define DEFAULT_DB_STORE_SIZE (1ULL << 40)/#define DEFAULT_DB_STORE_SIZE (1ULL << 32)/g' TrustedComputeFramework/tc/sgx/common/packages/db_store/lmdb_store.cpp &&\
+    cd /project/TrustedComputeFramework/tools/build &&\
+    echo "CLEAN BUILD..." && make clean && echo "DONE CLEAN" && make && echo "BUILD SUCCESS"
+
+WORKDIR /project/TrustedComputeFramework
```

Comment:
Add new line at end of file

Reply:
done as https://github.com/hyperledger-labs/trusted-compute-framework/pull/47/commits/17002ab20a329bdedaca59b9750c36c482756e71